lib/improver/cube_combiner.py

```diff
@@ -0,0 +1,174 @@
+# -*- coding: utf-8 -*-
```

Comment:
Could this file be in the utilities module?

Reply:
I did consider this but it has it's own CLI so I decided it should not be in the utilities module.